MIGRATE.md

```diff
@@ -34,49 +34,54 @@ What will be exported to RMT:
 
 * If you do choose to export your SSL certificate, please keep it safe.
 
-1. Copy over the `smt-data-export` script to your SMT server. The script is located at
-https://github.com/SUSE/rmt/blob/master/contrib/smt-to-rmt-helpers/smt-data-export or you can download it directly with
-the following command.
-    ```bash
-    wget https://raw.githubusercontent.com/SUSE/rmt/master/contrib/smt-to-rmt-helpers/smt-data-export
-    ```
-2. Make the script executable:
-    ```bash
-    chmod +x smt-data-export
-    ```
-3.
+1. Update your SMT server installation.
+
+2. Run the smt-data-export script which is available true the latest smt version:
```

Comment:
`is available true` (typo). I would specify the exact SMT version, i.e. `which is available in SMT versions >= 3.0.33`, which will be the next version.